Improve and refactor tutorial and usecase content (#3680)
wso2/docs-apim: en/docs/use-cases/streaming-tutorials/expose-a-kafka-topic-as-a-managed-websocket-api.md

 
## Step 11 - Results evaluation

> As SI server is already running as explained in above steps, Once the kafka client sends the events they will be consumed by the source in SI server and pushed to websocket server. 

Reviewer: ```suggestion
As SI server is already running as explained in above steps, when the Kafka client sends the events they will be consumed by the source in the SI server and pushed to the WebSocket server. 
```

> As WS client command listens to these events, below type of events will be printed in the terminal which WS client ran.

Reviewer: ```suggestion
As the WS client command listens to these events, the following types of events will be printed in the terminal that the WS client was run.
```
